INSTALL.md

```diff
@@ -108,11 +108,11 @@ configuration files included with find_package should set this up.
 
 ## Sub-Modules
 
-OpenEXR consists of four separate sub-modules - IlmBase, PyIlmBase,
-OpenEXR, OpenEXR_Viewers - which can be built independently. The
-repository’s top-level CMakeLists.txt defines a *super-project* that
-builds all four modules, and the steps above for running cmake at the
-top level of the repo build each of the sub-modules, in parallel.
+OpenEXR consists of three separate sub-modules - IlmBase, PyIlmBase,
+and OpenEXR - which can be built independently. The repository’s
+top-level CMakeLists.txt defines a *super-project* that builds all
+four modules, and the steps above for running cmake at the top level
```

Comment:
Shouldn't this then be `builds all three modules` or just `builds all modules`?

Reply:
Oops, good catch, thanks.